app/models/internal_post.rb

```diff
@@ -1,6 +1,8 @@
 class InternalPost < Post
   belongs_to :user
   validates :user, presence: true
+  validates :body_markdown, presence: true
```

Comment:
Feel free to merge









—
Sent from Mailbox

On Thu, Aug 6, 2015 at 11:41 AM, Karen Ventura <notifications@github.com>
wrote:

>> @@ -1,6 +1,8 @@
>>  class InternalPost < Post
>>    belongs_to :user
>>    validates :user, presence: true
>> +  validates :body_markdown, presence: true
> @FerPerales  now, it has a test for body_markdown
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/PowerCodeGirls/RubyGameDev.com/pull/7/files#r36436439